### PR TITLE
threedifier 1.0.3 (new formula)

### PR DESCRIPTION
**Pull request by Ylannl**
add new formula for the 3dfier software

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/threedifier.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Threedifier < Formula
+  desc "Takes 2D GIS datasets and 3dfies them"
+  homepage "https://github.com/Homebrew/homebrew-core"
+  url "https://github.com/tudelft3d/3dfier/archive/v1.0.3.tar.gz"
+  sha256 "0216cdb4bbad4f6ec8a3867742129db2b365a9f92a6cb6744bb6266be4f466f5"
```
Comment (lembacon) [inline Formula/threedifier.rb]: A blank line after `sha256`.

File: Formula/threedifier.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  depends_on "liblas"
+  depends_on "yaml-cpp"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "{bin}/3dfier", "--version"
```
Comment (lembacon) [inline Formula/threedifier.rb]: Please add a test that exercises a deeper level of the functionality provided by this formula.

**Review by lembacon — CHANGES_REQUESTED** [review]

File: Formula/threedifier.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+class Threedifier < Formula
+  desc "Takes 2D GIS datasets and 3dfies them"
+  homepage "https://github.com/Homebrew/homebrew-core"
```
Comment (fxcoudert) [inline Formula/threedifier.rb]: Homepage should probably be https://github.com/tudelft3d/3dfier

**Comment by fxcoudert** [comment]
```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
LASZIP_INCLUDE_DIR
   used as include directory in directory /tmp/threedifier-20180929-32481-vkt436/3dfier-1.0.3
```

**Comment by fxcoudert** [comment]
Please make sure the formula builds locally and passes `brew audit --strict` before pushing to the pull request. Most of the issues can be fixed by yourself, without requiring multiple rounds of review, which give us more time to work on other pull requests.

**Comment by Ylannl** [comment]
> Please make sure the formula builds locally and passes `brew audit --strict` before pushing to the pull request. Most of the issues can be fixed by yourself, without requiring multiple rounds of review, which give us more time to work on other pull requests.

I actually did this on my own system and it passed. But I'll look into the issue and fix it as well as the deeper test. Thanks!

**Comment by Ylannl** [comment]
I've decided to put this formula in a separate tap, together with some other packages. Thanks again.

**Pull request closed without merging**